docs/definitions.rst

```diff
@@ -12,11 +12,13 @@ to the given exceedance probability [1]_.
 **Exceedance probability**
 
 Probability that an environmental contour is exceeded (exceedance probability). The exceedance probability, α,
-corresponds to a certain recurrence or return period, T, which describes the average time period between two consecutive
-environmental states that exceed the contour . Note that exceedance can be defined in various ways for environmental
-contours (Mackay and Haselsteiner, 2021) [2]_ .
+corresponds to a certain recurrence or return period, Tr, which describes the average time period between two
```

Comment:
Please consider the following changes:
 * change "Here, the return.." to "In the following formula, the return..."
 * Delete the sentence "E.g. when considering.."
 * Move the sentence "Note that exceedance can ..." to appear directly after the formula instead of directly before the formula.